tests/de/gurkenlabs/litiengine/DirectionTests.java

```diff
@@ -34,50 +34,44 @@ public void fromAngle_Right(double angle) {
         assertEquals(Direction.RIGHT, Direction.fromAngle(angle));
     }
 
-    @Test
```

Comment:
Please remove the unused `import org.junit.jupiter.api.Test;`

Reply:
Removed the unused import